packages/muon/components/inputter/src/inputter-component.js

```diff
@@ -220,13 +238,11 @@ export class Inputter extends ScopedElementsMixin(ValidationMixin(MaskMixin(Muon
     return html`
       <div class="${classMap(this.classes)}" style="${styleMap(this.inlineStyles)}" aria-describedby=${ifDefined(this.helper && !this.__isTipDetailAvailable ? this._helperId : undefined)}
         aria-details=${ifDefined(this.helper && this.__isTipDetailAvailable ? this._helperId : undefined)}>
-        ${this._isMultiple ? this._addHeading : this._addLabel}
-        ${this._addHelper}
-        <div class="wrapper">
-          ${super.standardTemplate}
-          ${this._addMask}
-          ${this._addInputTypeIcon}
-        </div>
+        ${this._isMultiple ? html`
+            <fieldset>
+              ${this.__wrapperContent}
+            </fieldset>
+          ` : this.__wrapperContent}
```

Comment:
Slight tweak to whitespace

```suggestion
        ${this._isMultiple ? html`
          <fieldset>
            ${this.__wrapperContent}
          </fieldset>
        ` : this.__wrapperContent}
```